src/ThisAssembly.Resources/CSharp.sbntxt

```diff
@@ -0,0 +1,50 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//
+//     ThisAssembly.Constants: {{ Version }}
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+{{ func resource }}
+	    /// <summary>
+        {{~ if $0.Comment ~}}
+        /// {{ $0.Comment }}
+        {{~ else ~}}
+	    /// => @"{{ $0.Path }}"
+        {{~ end ~}}
+	    /// </summary>
+        public static partial class {{ $0.Name }}
+        {
+            {{~ if $0.IsText ~}}
+            private static string s_text;
```

Comment:
I have a serious dislike of prefixes and underscores on fields. Let's just call this `text`.

Reply:
Personal preferences - even in C#, the bane of existence. :) 

Another difference between us... private methods should have `private` marked on them, but that's an error in your repo. :)